packages/deck/backend/sirius-components-deck-graphql/pom.xml

```diff
@@ -40,33 +40,29 @@
 	</distributionManagement>
 
 	<dependencies>
-		<dependency>
-			<groupId>com.graphql-java</groupId>
-			<artifactId>graphql-java</artifactId>
-		</dependency>
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
 		</dependency>
 		<dependency>
 			<groupId>org.eclipse.sirius</groupId>
 			<artifactId>sirius-components-graphql-api</artifactId>
-            <version>2024.5.0</version>
+      <version>2024.5.0</version>
```

Comment:
There is something wrong with the indentation

Reply:
tab vs space, I've fixed that